processor/k8sattributesprocessor/README.md

```diff
@@ -82,6 +105,22 @@ Additional container level attributes can be extracted provided that certain res
    instance. If it's not set, the latest container instance will be used:
    - container.id (not added by default, has to be specified in `metadata`)
 
+Please note however that container level attributes can not be used for source rules in the pod_association.
+
+Example for extracting container level attributes:
+
+```yaml
```

Comment:
may be a trivial example, but it's always helpful especially for new users to see exactly where these properties need to be set. Can remove it though if it makes the readme too verbose